content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: rolldice
+weight: 10
+---
+
+This page will show you how to get started with OpenTelemetry in C++.
+
+You will learn how you can instrument a simple C++ application automatically, in
+such a way that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the console.
+
+## Prerequisites 
+
+Ensure that you have the following installed locally:
+
+- Git
+- C++ compiler supporting C++ version >= 11
+- Make 
+- CMake version >= 3.1
+- [Conan](https://conan.io/)
```

Comment:
@open-telemetry/cpp-approvers would you be OK with this getting started guide, making use of Conan? I remember that this was not something we (otel community) provide as a first class supported thing?

Reply:
cc @open-telemetry/cpp-approvers 

Some external packages are listed here:

https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md#using-package-managers

opentelemetry-cpp does not maintain packages in these package managers.

Taking conan for example, it is out of date at time of writing:

* The latest conan package is 1.9.1, depends on curl libraries which have a CVE, contains code that is now removed (jaeger exporter, thrift), still uses C++11 which has been removed.
* The latest opentelemetry-cpp version is 1.12.0

For a C++ Getting Started doc, I think it is best to use CMake with only the opentelemetry-cpp code itself, and stay away from third parties (from the otel point of view) package managers.

Reply:
Also, only C++ >= 14 is supported now.

Reply:
@Akhaled19 would it be possible to not use conan through out the documentation? This would mean end-users need to clone the oatpp and opentelemetry repos and then build things from source code themselves.

The Currency Service of the Demo may be a template for you as well, take a look here:

https://github.com/open-telemetry/opentelemetry-demo/tree/main/src/currencyservice